lib/edge-runtime/ai.ts

```diff
@@ -297,6 +298,8 @@ export async function Angela( // Good ol' Angela
         originalToPlaceholderMap,
       );
 
+      rawOutput = stripExampleFunctions(rawOutput);
```

Comment:
Could this live inside `streamResponseToUser()` instead? Keeps Angela function cleaner